formbricks/formbricks
# feat: display survey on generic actions

## What does this PR do?

This PR adds two more automatic actions, Exit Intent and Scroll Depth 50%

Fixes #391

![Screenshot](https://github.com/formbricks/formbricks/assets/62984427/a3d2e788-038f-43cb-a48a-7eb6639693c0)


## Type of change




## How should this be tested?



- Go to actions. here you can see these two automatic actions
- Open the demo app, you can check exit intent here and in order to check scroll depth 50%, we need to add more content in demo, which can be dummy lorem ipsum text

## Checklist
